grails-app/views/putAway/_print.gsp

```diff
@@ -124,9 +142,15 @@
                         <wordwrap:td>${putawayItem["inventoryItem.expirationDate"]}</wordwrap:td>
                         <wordwrap:td>${putawayItem?.quantity}</wordwrap:td>
                         <wordwrap:td>${putawayItem?.quantity}</wordwrap:td>
-                        <wordwrap:td>${(putawayItem["preferredBin.zoneName"] ? putawayItem["preferredBin.zoneName"] + ": " : '') + putawayItem["preferredBin.name"]}</wordwrap:td>
+                        <wordwrap:td>
+                            ${(putawayItem["preferredBin.zoneName"] ? putawayItem["preferredBin.zoneName"] + ": " : '') +
+                                    (putawayItem["preferredBin.name"] ?: g.message(code: "default.null.label", default: "null"))}
```

Comment:
I looked at the ticket and saw "null" in the list of things to translate.
![image](https://github.com/openboxes/openboxes/assets/1162935/0acfcb9d-5869-4049-ae47-1bba4adab186)

But please don't do this. This is the type of thing we need to either push back on or ask clarification questions about. I honestly think the inclusion of "null" is a bug that was never fixed. 

We have three options:
* Leave it blank (i.e. empty string) - my preference
* default.none.label
* default.na.label

If you want to get clarification from the team please request clarification in the ticket. Otherwise, move forward with the "leave it blank" option.

Reply:
@jmiranda I usually assume that the ticket is written **properly** and I don't have to overthink every single bullet of the ticket, and since it wasn't challenging to implement this, I did it, but @awalkowiak pointed that out and asked Kelsey about her opinion on that, and she felt ok with leaving it empty, which I will fix now.

Reply:
Moreover - while getting to this bullet, I let Katarzyna know, that the requirement feels weird to me, but I didn't get any reaction like "wtf", so I assumed that maybe I was weird that I complained about that :smile: